src/resources/elements/primeDesignSystem/pform-item/pform-item.html

```diff
@@ -0,0 +1,15 @@
+<template>
```

Comment:
I'm expecting to see a character counter in here.  Why do we need to call it "labelExtra"?  Can we just have a boolean bindable `noCounter` (default false) to indicate whether there should be a counter, and do the counting internal to this control by inspecting the value of the input?  It is OK to assume the slot contains a single input, even if a little nested like `numeric-input`.

Reply:
Keep in mind that this will require a `max-length` attribute in order to show how many characters you can write

Reply:
ok, i'll take your recommendation and go with Option 1.